src/test/resources/processor/IndexMappings.json

```diff
@@ -67,6 +67,13 @@
             }
           }
         }
+      },
+      "passage_embedding": {
+        "type": "knn_vector",
+        "dimension": 768
+      },
+      "passage_text": {
+        "type": "text"
```

Comment:
why we need these new fields? why we cannot use older fields?

Reply:
Because they don't have text matching

Reply:
do we need text matching?

Reply:
yeah i wrote a test case where I am passing querytext as hello world. If I don't add this field here then it will fail